src/components/Content/ParticipantList/ParticipantList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getUsers } from "../../../actions";
+import Participant from './Participant/Participant';
+import './Participant/Participant.css'
+
+class ParticipantList extends React.Component {
+  componentWillMount() {
+    const { getUsers } = this.props;
+    getUsers();
+  }
+
+  renderList() {
+    const { users } = this.props;
+
+    return Object.keys(users).sort().reduce((acc, key) => {
+      const { username } = users[key];
+      return acc.concat(<Participant key={username} username={username} />)
+    }, []);
```

Comment:
@wittnl Would you mind just turning this into a for-loop?  It'll be simpler for people to read, and also use less memory in virtue of not allocating a function on every iteration.